src/pages/index.js

```diff
@@ -42,17 +45,40 @@ const cardModalFormValidator = new FormValidator(
   validatorConfig,
   document.forms["card-modal-form"]
 );
+const avatarModalFormValidator = new FormValidator(
+  validatorConfig,
+  document.forms["avatar-modal-form"]
+);
 
 profileModalFormValidator.enableValidation();
 cardModalFormValidator.enableValidation();
+avatarModalFormValidator.enableValidation();
+
+/*******
+ * API *
+ *******/
+const token = "c2b0fb8f-5e2c-45d3-9e40-bbe48905b446";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: token,
+    "content-type": "application/json",
+  },
+});
 
 /************
  * SECTIONS *
  ************/
-const cardSection = new Section(
-  { items: initialCards, renderer: createNewCardEl },
-  ".cards__list"
-);
+let cardSection;
+
+api.getInitialCards().then((result) => {
+  cardSection = new Section(
+    { items: result, renderer: createNewCardEl },
+    ".cards__list"
+  );
+  cardSection.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**